gradle/versions.gradle

```diff
@@ -1,22 +1,23 @@
-/*
- * Base project version and versions of common dependencies.
- */
-
-version = '2.14.0-SNAPSHOT'
-
-ext.versions = [
-	'xtext': version,
-	'xtext_bootstrap': '2.13.0',
-	'gradle_plugins': '0.1.0',
-	'xtext_gradle_plugin': '1.0.20',
-	'emfEcore': '[2.10.2,3)',
-	'emfMwe': '1.3.21.201705291010',
-	'emfMwe2': '2.9.1.201705291010',
-	'jdtCore': '3.12.2',
-	'jdtCompilerApt': '1.2.100',
-	'xpand': '2.0.0',
-	'asm': '6.0',
-	'icu': '52.1',
-	'guice': '3.0',
-	'junit': '4.12'
-]
+/*
+ * Base project version and versions of common dependencies.
+ */
+
+version = '2.14.0-SNAPSHOT'
+
+ext.versions = [
+	'xtext': version,
+	'xtext_bootstrap': '2.13.0',
+	'gradle_plugins': '0.1.0',
+	'xtext_gradle_plugin': '1.0.20',
+	'emfEcore': '[2.10.2,3)',
+	'emfMwe': '1.3.21.201705291010',
```

Comment:
Do we need an MWE version with asm 6 here, too?

Reply:
that is not yet built :(